Makefile

```diff
@@ -62,7 +62,7 @@ else
 endif
 
 functional-tests:
-	pytest --gherkin-terminal-reporter tests -vvv --cucumberjson=test-report.json --durations=0 --html=test-report.html
+	docker compose run --env UMBRELLA_PATH=. --rm rstuf-ft-runner bash tests/functional/scripts/run-ft-das-all.sh $(CLI_VERSION) $(SLOW)
```

Comment:
Why did you decide to change the `functional-tests` instead of adding `functional-tests-all`?
This can have unintended consequences.
I suggest you add a new:
`functional-tests-all` makes the command more verbose and clearer about what is happening.

Reply:
@MVrachev because they were not working. The functional tests expect to run in the Docker compose network, calling for example http://repository-service-tuf-api, so running them locally doesn't work, unless everything else is setup locally.
The command here was moved to this place instead and runs after the ceremony happens: https://github.com/repository-service-tuf/repository-service-tuf/blob/d7e3ba6eff1e97b80f9179718fa03361267c639f/tests/functional/scripts/run-ft-das-all.sh#L36 
 
 But I can do like you suggest and leave the two options.